<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementWidget</class>
 <widget class="QWidget" name="MeasurementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Measurement</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2D Measurement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="measurement2DTableWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Lumen Area / mm^2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Vessel Wall Area / mm^2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Wall Thickness / mm</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>NMI</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Tract 1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tract 2</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3D Measurement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="measurement3DTableWidget">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <row>
         <property name="text">
          <string>Total Plaque /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Lumen /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Vessel Wall /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Calcification /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Hemorrhage /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LRNC /mm^3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LM /mm^3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Volume</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Stenosis %</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="stenosisSpinBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>label_2</zorder>
     <zorder>measurement2DTableWidget</zorder>
     <zorder>label_3</zorder>
     <zorder>measurement3DTableWidget</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Generate Report</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
